Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift

```diff
@@ -9,34 +9,44 @@ import UIKit
 
 class HomeLogoHeaderCell: UICollectionViewCell, ReusableCell {
     private struct UX {
+        static let iPadAdjustment: CGFloat = -40
         struct Logo {
-            static let imageSize: CGFloat = 40
-            static let topConstant: CGFloat = 32
+            static let iPhoneImageSize: CGFloat = 40
+            static let iPadImageSize: CGFloat = 75
+            static let iPhoneTopConstant: CGFloat = 32
+            static let iPadTopConstant: CGFloat = 70
             static let bottomConstant: CGFloat = -10
         }
 
         struct TextImage {
-            static let imageWidth: CGFloat = 70
-            static let imageHeight: CGFloat = 40
-            static let leadingConstant: CGFloat = 9
+            static let iPhoneWidth: CGFloat = 70
+            static let iPadWidth: CGFloat = 133
+            static let iPhoneLeadingConstant: CGFloat = 9
+            static let iPadLeadingConstant: CGFloat = 17
             static let trailingConstant: CGFloat = -15
         }
     }
 
     typealias a11y = AccessibilityIdentifiers.FirefoxHomepage.OtherButtons
 
     // MARK: - UI Elements
-    lazy var logoImage: UIImageView = .build { imageView in
+    private lazy var logoImage: UIImageView = .build { imageView in
         imageView.image = UIImage(imageLiteralResourceName: ImageIdentifiers.homeHeaderLogoBall)
         imageView.contentMode = .scaleAspectFit
         imageView.accessibilityIdentifier = a11y.logoImage
     }
 
-    lazy var logoTextImage: UIImageView = .build { imageView in
+    private lazy var logoTextImage: UIImageView = .build { imageView in
         imageView.contentMode = .scaleAspectFit
         imageView.accessibilityIdentifier = a11y.logoText
     }
 
+    private lazy var containerView: UIView = .build { view in
+        view.backgroundColor = .clear
+        view.accessibilityLabel = AppName.shortName.rawValue
```

Comment:
Should we? It's only accessible in the sense that it has voice over capability, but it's not accessible for the end-user otherwise, and the container itself doesn't need to be identified for anything, does it?

Reply:
Also just thought, if we want this further testable with UI tests, then we could have a different a11y identifier when we have the private mode image VS normal mode image. I think it's worth asking Clare her opinion on this